===> Checking for items in STAGEDIR missing from pkg-plist
Error: Orphaned: bin/libSvtHevcEnc.so
Error: Orphaned: include/EbApi.h
Error: Orphaned: include/EbErrorCodes.h
Error: Orphaned: include/EbTime.h

--- source/CMakeLists.txt.orig	2019-01-23 09:47:18 UTC
+++ source/CMakeLists.txt
@@ -570,13 +570,6 @@ if(ENABLE_HDR10_PLUS)
         ARCHIVE DESTINATION ${LIB_INSTALL_DIR})
 endif()
 
-if(SVTHEVC_FOUND)
-    install(FILES "${SVT_HEVC_INCLUDE_DIR}/EbApi.h" DESTINATION include)
-    install(FILES "${SVT_HEVC_INCLUDE_DIR}/EbErrorCodes.h" DESTINATION include)
-    install(FILES "${SVT_HEVC_INCLUDE_DIR}/EbTime.h" DESTINATION include)
-    install(FILES "${SVT_HEVC_LIBRARY}" DESTINATION ${BIN_INSTALL_DIR})
-endif()
-
 install(FILES x265.h "${PROJECT_BINARY_DIR}/x265_config.h" DESTINATION include)
 if((WIN32 AND ENABLE_CLI) OR (WIN32 AND ENABLE_SHARED))
     if(MSVC_IDE)
